.github/scripts/setup.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -e
 export DEBIAN_FRONTEND=noninteractive
-export BITCOIN_VERSION=0.20.1
+export BITCOIN_VERSION=24.0
```

Comment:
ah this wont wont because the full url is https://bitcoincore.org/bin/bitcoin-core-24.0/test.rc4/bitcoin-24.0rc4-x86_64-linux-gnu.tar.gz. Probably easiest to just append rc4 after the second BITCOIN_VERSION for now.